recipe/meta.yaml

```diff
@@ -33,19 +33,19 @@ requirements:
         - xerces-c
         - libnetcdf
         - krb5
-        - libtiff
+        - libtiff 4*
         - libpng
         - openjpeg
         - curl
         - expat
         - zlib  # [not win]
-        - jpeg  # [not win]
+        - jpeg 9*  # [not win]
         - postgresql  # [linux]
         - jasper  # [not win]
         - giflib  # [not win]
         - json-c  # [not win]
         - freexl  # [not win]
-        - xz  # [not win]
+        - xz 5*  # [not win]
```

Comment:
If we pin `xz 5*` Python 3.5 will downgrade to `xz 5.0.5` and that is a no-no in OS X. (See the problem you are having with Pyllow. I had that same issue in many other packages.)

If using `xz 5.2*` we are OK in OS X, but then we get (on Python 3.5):

```shell
Error: The following specifications were found to be in conflict:
  - numpy 1.11* -> python 3.5*
  - python 3.5* -> xz 5.0.5
  - setuptools -> python 3.5*
  - xz 5.2*
```